sql/core/src/main/scala/org/apache/spark/sql/jdbc/JdbcDialects.scala

```diff
@@ -229,6 +229,29 @@ abstract class JdbcDialect extends Serializable with Logging{
     }
   }
 
+  /**
+   * Check namespace exists or not.
+   */
+  def namespacesExists(conn: Connection, options: JDBCOptions, namespace: String): Boolean = {
```

Comment:
Under the JDBC context, we should say schema instead of namespace.

Reply:
OK